### ISSUE TITLE
Virtualbox fails to build with 5.11 kernel

### ISSUE THREAD

**Issue by bryanasdev000** [NixOS/nixpkgs#114949]
**Describe the bug**

Virtualbox fails to build with current unstable kernel 5.11, upstream already have some tickets opened for this at https://www.virtualbox.org/ticket/20198 .

**To Reproduce**
1. Use boot.kernelPackages = pkgs.linuxPackages_zen; # Or 5.11 equivalent
2. Set virtualisation.virtualbox.host.enable = true;
3. Try to nixos-rebuild

**Expected behavior**

Virtualbox build and works.

**Screenshots**

N/A

**Additional context**

N/A

**Notify maintainers**

@sander

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.15-zen2, NixOS, 21.05.20210224.f6b5bfd (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20201205_a5d85d0`
 - channels(root): `"nixos-21.03pre268835.8c8731330b5, home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
 ```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: virtualbox
# a list of nixos modules affected by the problem
module: virtualbox
```

**Comment by bryanasdev000** [comment]
I will try to test any available patches.

Also, the speed that Virtualbox breaks is fun: P

**Comment by bryanasdev000** [comment]
I will close this since @andresilva fixed with 44993187127edd1e0a20dbe03bcf5bea22943d74 .
